src/InteractiveWindow/EditorTest/InteractiveWindowTests.cs

```diff
@@ -1295,6 +1295,215 @@ public void PasteWithSelectionInReadonlyArea()
             AssertCaretVirtualPosition(2, 4);             
         }
 
+        [Fact]
+        public void DeleteLineWithOutSelection()
+        {
+            Submit(
+@"1",
+@"1
+");                                                                                                                        
+            var caret = Window.TextView.Caret;                               
+
+            // DeleteLine with caret in readonly area
+            caret.MoveToPreviousCaretPosition();
+            caret.MoveToPreviousCaretPosition();
+            caret.MoveToPreviousCaretPosition();
+
+            AssertCaretVirtualPosition(1, 1);
+            Task.Run(() => Window.Operations.DeleteLine()).PumpingWait();
```

Comment:
This is something I don't understand actually. What operation need to happen on a separate thread?